packages/visx-brush/src/BrushHandle.tsx

```diff
@@ -160,21 +181,38 @@ export default class BrushHandle extends React.Component<BrushHandleProps> {
                 onPointerLeave={isControlled ? undefined : dragEnd}
               />
             )}
-            <rect
-              x={x}
-              y={y}
-              width={width}
-              height={height}
-              fill="transparent"
-              className={`visx-brush-handle-${type}`}
-              onPointerDown={dragStart}
-              onPointerMove={dragMove}
-              onPointerUp={isControlled ? undefined : dragEnd}
-              style={{
-                cursor,
-                pointerEvents: !!brush.activeHandle || !!brush.isBrushing ? 'none' : 'all',
-              }}
-            />
+            {!renderBrushHandles && (
+              <rect
+                x={x}
+                y={y}
+                width={width}
+                height={height}
+                fill="transparent"
+                className={`visx-brush-handle-${type}`}
+                onPointerDown={dragStart}
+                onPointerMove={dragMove}
+                onPointerUp={isControlled ? undefined : dragEnd}
+                style={{
+                  cursor,
+                  pointerEvents: !!brush.activeHandle || !!brush.isBrushing ? 'none' : 'all',
+                }}
+              />
+            )}
+            {renderBrushHandles && (
+              <g
+                onPointerDown={dragStart}
+                onPointerMove={dragMove}
+                onPointerUp={isControlled ? undefined : dragEnd}
+              >
+                {renderBrushHandles({
+                  ...this.props.handle,
+                  height: stageHeight,
+                  className: `visx-brush-handle-${type}`,
+
+                  isBrushActive: brush.extent.x0 !== -1 && brush.extent.x1 !== -1,
```

Comment:
does it make sense to use `brush.isBrushing` here instead of using different logic?

Reply:
ah I misinterpreted this. it's not whether brushing is happening, it's whether the handle is used as a trigger area?? is this necessary given that only `BrushHandles` for the specified `resizeTriggerAreas` are rendered?

https://github.com/airbnb/visx/blob/master/packages/visx-brush/src/BaseBrush.tsx#L635

Reply:
Thanks for the read @williaster I really appreciate it. 
The `isBrushActive` function is just for cosmetics - hide the handles when there is no area selected, otherwise the handles are rendered at `x=0` position. 
Is there a better way to do this or should this be handled by the user?